2021-06-14-announcements.md

```diff
@@ -0,0 +1,41 @@
+# CFDE Announcements
+
+## Cross-Pollination Events
+The next scheduled meeting is **Tuesday, July 6th**. Regular Cross-Pollination events take place on the first Tuesday of every month. Please check out the [Cross-Pollination Schedule](https://docs.google.com/spreadsheets/d/1hQAeOLkivUZZnwZ_KxfGw3neezMaWbrPk9nnFiKfQGA/edit?usp=sharing) and let us know if you would like to present! Please contact us today if you are interested in presenting at support@cfde.atlassian.net.
+
+## CFDE Trainings and Workshops
+
+On **June 16th** at **10am PDT**, join us and the LINCS Project for a 1-hour demonstration of using Appyters, which are web-based customizable Jupyter notebooks executed in the cloud, to generate downstream visualization and enrichment analysis from a list of differentially expressed genes.  Appyters can be applied to a variety of workflows including building customized machine learning pipelines, analyzing omics data, and producing publishable figures. [Click Here to Register](https://www.nih-cfde.org/events/pop-up-series-introduction-to-rnaseq-related-appyters/?pk_campaign=anc).
+
+On **June 21st and 23rd** we will host a two part (2 hour sessions each) workshop series that will provide hands-on experience in conducting basic RNA-Seq analysis in the cloud. We’ll go over how to process RNA-Seq data and cover the basics of differential gene expression and data visualization in R. This workshop will be conducted remotely via Zoom. The workshop Zoom link will be emailed to participants after registration closes. Participants should plan to fill out a brief survey before and after the workshop. Register here:
+June 21st: [RNA-Seq in the Cloud Part I](https://www.nih-cfde.org/events/rnaseq-in-the-cloud-2021-06-21/?pk_campaign=anc)
+June 22nd: [RNA-Seq in the Cloud Part II](https://www.nih-cfde.org/events/rnaseq-in-the-cloud-2021-06-23/?pk_campaign=anc) 
```

Comment:
June 23rd

Reply:
Thanks Marisa. I think since the date is right in the description and the link works I'm going to let that one go rather than sending out an update. :(